src/submission/exceptions/exceptions.ts

```diff
@@ -75,7 +82,13 @@ const validateFieldValueWithExceptions = ({
           exception.requested_core_field === validationErrorFieldName &&
           exception.schema === schemaName,
       )
-      .forEach(matchingException => allowedValues.add(matchingException.requested_exception_value));
+      .forEach(matchingException => {
+        if (isNumericField) {
+          allowedValues.add(undefined);
```

Comment:
Not sure if '' (empty string) should be included here as well

Reply:
`undefined` is the correct value here. If we allow `''` we also need to convert it into `undefined` eventually. The way this code is strucutred, i think the value normalization happens before this validation code so we would want that conversion from empty string to `undefined` to happen outside of this function.